selene/src/json_output.rs

```diff
@@ -79,3 +90,24 @@ pub fn diagnostic_to_json(
             .collect(),
     }
 }
+
+pub fn log_total_json(
+    mut stdout: StandardStream,
+    parse_errors: usize,
+    lint_errors: usize,
+    lint_warnings: usize,
+) -> io::Result<()> {
+    writeln!(
+        stdout,
+        "{}",
+        serde_json::to_string(&JsonOutput::Summary(JsonSummary {
+            errors: lint_errors,
+            warnings: lint_warnings,
+            parse_errors
+        }))
+            .unwrap()

```

Comment:
If we're going to be returning a Result here anyway, these `.unwrap()` should be `?`